### PR TITLE
change the style of the lock information in the complaints view to av…

### LINKED ISSUE #1383: Non existing lock presented on complaint view
### Describe the bug
If you assess a complaint on the top right corner will be displayed who had the lock for the original assessment, the complaint function ifself has no lock feature though.

#### To Reproduce
1. Submit something (e.g. a Text Exercise) with P1
2. Assess it with P2
3. Complain on this assessment with P1
4. Assess the complaint with P3
5. Have a look on the top right corner. This should state "Locked by P2" or something

#### Expected behavior
There is no lock message or there is something stated, maybe not in red, like "Assessment by P2"
"Lock by ..." makes people think there is a lock on the complaint which they don't have access on.

In addition: if you are the Person that was complained about it shoud probably explain why you don't have access to it (beeing able to update or reject) to prevent confusion.

#### Additional Information ####
4405acb8492eb4f21150755efd9b4596
This is not an active complaint but a resolved one but the view is the same.
I blacked out the name of the tutor who assessed the submission originally, the submission itself and the grading criteria.

You can see the lock message on the top right corner. It relates only to the assessment itself, bit is quite misleading in my opinion.

### Environment

<details><pre>
 - OS: Windows Pro
 - Browser: Chrome v81.0.4044.129
</pre></details>

### PR DESCRIPTION
**Pull request by HanyaElhashemy**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I added multiple screenshots/screencasts of my UI changes
- [x] Client: I translated all the newly inserted strings into German and English

### Motivation and Context

resolve #1383 
### Description

change the color of the Lock information to black and change the Information from "Locked by..." to "original assessor is..." to avoid confusion
### Steps for Testing

1.Click on evaluate complaint and check that the lock information is present and avoids misunderstandings

### Screenshots

image

### CODE REVIEW

File: src/main/webapp/app/assessment/assessment-header/assessment-header.component.html [hunk 1]
```diff
@@ -8,14 +8,23 @@ <h3 class="top-container">
...
         </span>
+        <span
+            *ngIf="!isAssessor && isComplaint"
+            class="ml-3"
+            style="font-size: 65%;"
+            jhiTranslate="artemisApp.assessment.assessmentLockedComplaintView"
+            [translateValues]="{ otherUser: result?.assessor?.name + ' (' + result?.assessor?.login + ')' }"
+        >
+            Assessment locked by another user!
+        </span>
```
Comment (sascha11110) [inline src/main/webapp/app/assessment/assessment-header/assessment-header.component.html:27]: I think you can merge this lines and prevent a lot of duplicated code as you are using the `*ngIf` only to differentiate the translation strings. You can place the conditional operator inline with the translation string (`{{ (isComplaint ? 'artemisApp.assessment.assessmentLockedComplaintView' : 'artemisApp.assessment.assessmentLocked') | translate}}`) and keep `*ngIf="!isAssessor"` on the `span`element.

If I see this correctly, the id (`id="assessmentLocked"`) is not used anywhere either. So you can remove it.

Comment (HanyaElhashemy) [reply src/main/webapp/app/assessment/assessment-header/assessment-header.component.html:27]: Hmm.. and what about the class ? it is different, so I will have to use a conditional operator there as well. I thought is might be more readable to have another span element than putting conditional statements for each value.

Comment (sascha11110) [reply src/main/webapp/app/assessment/assessment-header/assessment-header.component.html:27]: Sorry, haven't seen that... was a little late.

File: src/main/webapp/app/assessment/assessment-header/assessment-header.component.html [hunk 2]
```diff
@@ -8,14 +8,23 @@ <h3 class="top-container">
...
             Assessment locked by another user!
         </span>
+        <span
+            *ngIf="!isAssessor && isComplaint"
+            class="ml-3"
+            style="font-size: 65%;"
+            jhiTranslate="artemisApp.assessment.assessmentLockedComplaintView"
+            [translateValues]="{ otherUser: result?.assessor?.name + ' (' + result?.assessor?.login + ')' }"
+        >
+            Assessment locked by another user!
```
Comment (krusche) [inline src/main/webapp/app/assessment/assessment-header/assessment-header.component.html:26]:
```suggestion
            Original Assessor: <name> <username>
```

File: src/main/webapp/app/assessment/assessment-header/assessment-header.component.html [hunk 3]
```diff
@@ -8,14 +8,23 @@ <h3 class="top-container">
...
         <span
             id="assessmentLocked"
-            *ngIf="!isAssessor"
+            *ngIf="!isAssessor && !isComplaint"
             class="text-danger ml-3"
             style="font-size: 65%;"
             jhiTranslate="artemisApp.assessment.assessmentLocked"
             [translateValues]="{ otherUser: result?.assessor?.name + ' (' + result?.assessor?.login + ')' }"
         >
             Assessment locked by another user!
```
Comment (krusche) [inline src/main/webapp/app/assessment/assessment-header/assessment-header.component.html:17]:
```suggestion
            Assessment locked by: otherUser
```

**Pull request merged**